faq/deploy-and-maintain-faq.md

```diff
@@ -146,3 +146,9 @@ Direct 模式就是把写入请求直接封装成 I/O 指令发到磁盘，这
     ```bash
     ./fio -ioengine=psync -bs=32k -fdatasync=1 -thread -rw=randrw -percentage_random=100,0 -size=10G -filename=fio_randread_write_test.txt -name='fio mixed randread and sequential write test' -iodepth=4 -runtime=60 -numjobs=4 -group_reporting --output-format=json --output=fio_randread_write_test.json
     ```
+
+## TiDB 支持在公有云上部署吗？
```

Comment:
Please help confirm whether the content for this item is outdated. Rest LGTM.

Reply:
OK

Reply:
It may take some time for the PM to confirm the information. I'll merge this PR first. If the PM confirms any outdated information, I'll create a separate PR to update it.